modules/snooker/src/main/ts/ephox/snooker/model/RunOperation.ts

```diff
@@ -40,10 +41,12 @@ export interface TargetUnmergable {
   readonly unmergable: () => Option<Element[]>;
 }
 
-export interface TargetPaste {
+// combines the above 4 interfaces because this is what data we actually get from TinyMCE
+export interface CombinedTargets {
```

Comment:
This is duplication of all the types (and the Option<any> appear to be incorrect). So can we remove that by just doing something like this?
```
export interface CombinedTargets extends TargetElement, TargetSelection, TargetMergable, TargetUnmergable { }
```

Reply:
yeah, was wondering if that'd make sense. will do!